docs/sources/flow/reference/components/otelcol.processor.probabilistic_sampler.md

```diff
@@ -0,0 +1,139 @@
+---
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.processor.probabilistic_sampler/
+labels:
+  stage: opensource
+title: otelcol.processor.probabilistic_sampler
+---
+
+# otelcol.processor.probabilistic_sampler
+
+`otelcol.processor.probabilistic_sampler` accepts logs and traces data from other otelcol components and apply probabilistic sampling based on configuration options.
+
+> **Note**: `otelcol.processor.probabilistic_sampler` is a wrapper over the upstream
+> OpenTelemetry Collector Contrib `probabilistic_sampler` processor. Bug reports or feature
+> requests will be redirected to the upstream repository, if necessary.
+
+You can specify multiple `otelcol.processor.probabilistic_sampler` components by giving them
+different labels.
+
+## Usage
+
+```river
+otelcol.processor.probabilistic_sampler "LABEL" {
+  output {
+    logs    = [...]
+    traces  = [...]
+  }
+}
+```
+
+## Usage for traces
+
+The `probabilistic_sampler` supports two types of sampling for traces: 
+1. `sampling.priority` [semantic
+   convention](https://github.com/opentracing/specification/blob/master/semantic_conventions.md#span-tags-table) as defined by OpenTracing
+2. Trace ID hashing
+
+The `sampling.priority` semantic convention takes priority over trace ID hashing. 
+Trace ID hashing samples based on hash values determined by trace IDs.
+
+### Arguments
+
+For traces `otelcol.processor.probabilistic_sampler` supports the following arguments:
+
+Name | Type     | Description | Default | Required
+---- |----------| ----------- |---------| --------
+`hash_seed`               | `uint32` | An integer used to compute the hash algorithm. Note that all collectors for a given tier (e.g. behind the same load balancer) should have the same hash_seed. |         | no
+`sampling_percentage`     | `float32`| Percentage at which traces are sampled; >= 100 samples all traces | `0`     | no
```

Comment:
Same here - shouldn't `sampling_percentage` be required?

Reply:
no, the default value is 0 = no sampling